### ISSUE TITLE
Update extract_2d documentation

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#7654]
_Issue [JP-3254](https://jira.stsci.edu/browse/JP-3254) was created on JIRA by [Nadia Dencheva](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dencheva):_

The extract_2d documentation on RTD for WFSS mode is confusing.

From a user
```java
The input_model I assume is the output from running AssignWcsStep on the corresponding direct image, but if I run the next example block to get the wavelengthrange file 1) I get {'wavelengthrange': 'N/A'} and 2) why is there a nircam example in the first box and a niriss wavelength range file in the second box?In the 3rd code box

“im” is called, but that isn’t defined anywhere. Is that supposed to be the datamodel aka “input_model”? Or is that a new datamodel of the dispersed image?I get an error that wfss_mmag_extract isn’t an argument anymore -> it looks like this is now called mmag_extractAfter all of that I get an error that ValueError: No source catalog listed in datamodel. how/where do I specify the source catalog? ```

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-3254?focusedCommentId=649946#comment-649946):_

[Nadia Dencheva](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dencheva) Do you recall if this came in via Help Desk or a more direct line of communication (slack, email, ...)?

**Comment by stscijgbot-jp** [comment]
_Comment by [Nadia Dencheva](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dencheva) on [JIRA](https://jira.stsci.edu/browse/JP-3254?focusedCommentId=649949#comment-649949):_

This came from INS through Slack.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-3254?focusedCommentId=650753#comment-650753):_

The RTD sections in question are here: <https://jwst-pipeline.readthedocs.io/en/latest/jwst/extract_2d/main.html#wfss-examples> which give examples of how someone could use the `assign_wcs` and `extract_2d` steps to create and process a custom list of grism objects for a given WFSS grism exposure.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-3254?focusedCommentId=650934#comment-650934):_

Fixed by #7684
